Python_Scripts/script_regroup_models.py

```diff
@@ -0,0 +1,337 @@
+"""
+Imports the SED-ML models and the SBML models from Biomodels and regroups them
+according to model properties (state variables, reactions, model name)
+in a newly created folder structure.
+One folder then corresponds to one benchmark model, with possibly multiple SBML
+files belonging to one benchmark model.
+"""
+
+# Note: libsedml must be imported before libsbml for whatever reason
+
+import libsedml
+import libsbml
+import os
+import shutil
+import pandas as pd
+import re
+
+from C import (DIR_MODELS_SEDML, DIR_MODELS_REGROUPED, DIR_MODELS, DIR_BASE)
+from setTime_BioModels import timePointsBioModels
+
+
+# get all models
+def regroup_models():
+    # get all models, list information about them
+    list_directory_sedml = sorted(os.listdir(DIR_MODELS_SEDML))
+    model_info = []
+
+    for sedml_model in list_directory_sedml:
+        # get the model name and year
+        model_name, model_year = _parse_model_name(sedml_model)
+        if model_name is None or model_year is None:
```

Comment:
short comment on what is happening here?

Reply:
good idea. Please be picky about doc.
At some point, I just wanted this stuff to run and started being lazy in documenting, due to some time pressure... Now it's time to document

Reply:
so models that don't have both name and year are ignored? does that happen for any?

Reply:
It happens once. That is logged, however.